not-so-trivial-pursuit/client/src/components/question-render.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from "react";
+
+const RadioButton = ({answer, index, onValueChange}) => {
+ let id = `${index}-choice`;
+  return( 
+  <div>
+    <input type="radio" id={id} name="answer" value={answer} onChange={onValueChange}/>
+    <label for={id}>{answer}</label>
+  </div>  
+)};
+
+const QuestionRender = ({question}) => {
+  const[isRight, setIsRight] = useState(false);
+  const handleSubmit = (evt) => {
+    evt.preventDefault();
+    console.log(evt.target);
+}
+  const onValueChange = (evt) => {
+    console.log(evt.target.value);
+    console.log(evt.target.value===question.correct_answer);
+    if(evt.target.value===question.correct_answer){
+      setIsRight(true);
+    }
+  }
+  return (
+    question && (
+      <div className={"question-section"}>
+      <div className="question-text">{question.question}</div>
+      <div className="answer-section">
+        <form onSubmit={handleSubmit}>
+          <RadioButton answer={question.incorrect_answers[0]} onValueChange={onValueChange}/>
+          <RadioButton answer={question.incorrect_answers[1]} onValueChange={onValueChange}/>
+          <RadioButton answer={question.correct_answer} onValueChange={onValueChange}/>
+          <RadioButton answer={question.incorrect_answers[2]} onValueChange={onValueChange}/>
+          <button type="submit">Submit Answer</button>
+        </form>
+      </div>
+      <div>
+        {isRight && "Yay! Correct!" }
```

Comment:
I think you could use a ternary operator here, like `{isRight ? "Yay! Correct!" : "Sorry, wrong answer"}`

Reply:
I originally did do that bit it was showing my wrong answer thing always because I didn't have the state set right.